src/agents/queenAgent.py

```diff
@@ -0,0 +1,12 @@
+from mesa import Agent
+
+class Queen(Agent):
+    def __init__(self, current_id, model, pos, id_anthill):
+        super().__init__(current_id, model)
+        self.pos = pos
+        self.id_anthill = id_anthill
```

Comment:
Existe a necessidade de salvar esse id se você já ta passando a model do Anthill ?
Pelo que eu percebi lendo aqui, esse é o grupo das formigas né ? Se for, é só usar a posição da casa (se baseia nas foragingsAnts)